lib/gds_api/govuk_headers.rb

```diff
@@ -0,0 +1,14 @@
+module GdsApi
+  class GovukHeaders
+    class << self
+      def headers
+        Thread.current[:headers] ||= {}
+      end
+
+      def values
```

Comment:
I don't think this method name adequately communicate what the method returns. Maybe the interface for this class would be more user-friendly if it was something like this:

```ruby
GdsApi::GovukHeaders.set_header(header_name, value) # sets the header header_name with value
GdsApi::GovukHeaders.headers # returns the headers that have been set
```

i.e. an explicit setter method for setting a given header rather than accessing the thread-local variable directly and a `headers` method that returns the cleaned up headers that have actual values.